assets/css/style.css

```diff
@@ -105,15 +109,59 @@ nav .nav-links-list li a {
     width: 80%;
   }
 
-  .nav-links {
-    display: none;
+  .nav-links-list {
+    margin-top: 1px;
+    position: absolute;
+    right: 0;
+    height: 90vh;
+    top: 8vh;
+    margin-left: 0;
+    background-color: #3e3c3c;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    transform: translateX(100%);
+    transition: transform 0.5s ease-in;
+    padding: 100px 0;
+  }
+
+  .nav-links-list li {
+    opacity: 0;
```

Comment:
- [ ] Kindly remove the opacity: 0 for the menu items in the mobile size as they don't appear.
![Screenshot from 2022-05-10 23-55-08](https://user-images.githubusercontent.com/33752735/167728465-a3cc5ee6-85be-4d51-a2b2-db098dacc7e6.png)